website/docs/r/rde_interface.html.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_interface"
+sidebar_current: "docs-vcd-resource-rde-interface"
+description: |-
+   Provides the capability of creating, updating, and deleting Runtime Defined Entity Interfaces in VMware Cloud Director.
+---
+
+# vcd\_rde\_interface
+
+Provides the capability of creating, updating, and deleting Runtime Defined Entity Interfaces in VMware Cloud Director.
+
+A Runtime Defined Entity Interface is specified unequivocally by 3 elements: `vendor`, `nss` and `version`. This
+3-tuple must be unique. See the examples section for more details.
+
+-> Creating, updating and deleting RDE Interfaces requires System administrator privileges.
+
+Supported in provider *v3.9+*
+
+## Example Usage
+
+```hcl
+resource "vcd_rde_interface" "my_interface1" {
+  vendor  = "bigcorp"
+  nss     = "tech"
+  version = "1.2.3"
+  name    = "BigCorp Interface"
+}
+
+resource "vcd_rde_interface" "my_interface2" {
+  vendor  = "bigcorp"
+  nss     = "tech"
+  version = "1.2.4"
+  name    = "Another BigCorp Interface"
+}
+```
+
+The second interface is valid because the version is different. If we set `version = "1.2.3"` in `my_interface2`,
+creation would fail despite having a different name, as the 3-tuple `vendor`, `nss` and `version` would be the same.
+
+## Argument Reference
+
+The following arguments are supported:
+
+-> The 3-tuple of `vendor`, `nss` and `version` specifies a unique RDE Interface.
+
+* `vendor` - (Required) The vendor of the RDE Interface.
+* `nss` - (Required) A unique namespace associated with the RDE Interface.
+* `version` - (Required) The version of the RDE Interface. Must follow [semantic versioning](https://semver.org/) syntax.
+* `name` - (Required) The name of the RDE Interface.
+
+## Attribute Reference
+
+* `readonly` - Specifies if the RDE Interface can be only read.
+
+## Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing RDE Interface can be [imported][docs-import] into this resource via supplying its vendor, nss and version, which
```

Comment:
Nit, but could we say namespace in this context? Or maybe put the variables in `backticks`?

Reply:
Makes sense, as I used backticks just a few paragraphs above.
I'd change it to namespace, but as the parameter is `nss` I'll just added the backticks.